### PR TITLE
Added a bunch of text and explanations for functions

### PR DESCRIPTION
**Pull request by DannyBoyG**
Changed "double" to "multiplier" because it makes more sense

Misc. small changes

### CODE REVIEW

**Comment by DannyBoyG** [comment]
Btw the "double" variable is now "multiplier". It multiplies the number of times that a set of random boats is placed. It always equals 1 unless the board is 32x32, in which case it equals 4

File: battleship.py [hunk 1]
```diff
@@ -10,40 +10,40 @@
 # sounds
 
 
-def hit_music():
+def hit_music():  # Plays serious music
```
Comment (RustyRaptor) [inline battleship.py:13]: Super

File: battleship.py [hunk 2]
```diff
@@ -10,40 +10,40 @@
...
 
-def skip_music():
+def skip_music():  # Plays recording of SammyOriginalSonicFan
     pygame.mixer.music.load('fricks.mp3')
     pygame.mixer.music.play(1)
 
 
-def hit_sound():
+def hit_sound():  # Plays a missile and explosion sound
     sound = pygame.mixer.Sound('388528__eflexmusic__artillery-explosion-close-mixed.wav')
```
Comment (RustyRaptor) [inline battleship.py:29]: Don't forget to change the filenames of these sounds so it's not taking up so much space in the code.

File: battleship.py [hunk 3]
```diff
@@ -58,112 +58,115 @@ def retrieve_size():  # Asks user for a difficulty and outputs length of one sid
 
 def clear_grid():  # Produces empty grid in the size chosen by user
     new_grid = []
-    for i in range(grid_size):
+    for i in range(grid_size):  # Fills new_grid list with empty lists
         new_grid.append([])
-    for i, k in enumerate(new_grid):
+    for i, k in enumerate(new_grid):  # Fills empty lists with "-", which is an empty space in-game
```
Comment (RustyRaptor) [inline battleship.py:63]: Super

File: battleship.py [hunk 4]
```diff
@@ -58,112 +58,115 @@ def retrieve_size():  # Asks user for a difficulty and outputs length of one sid
...
 
 
 def check_ships(ask, ships):  # Checks whether it's safe to place a ship in the coordinate ask
     if ask == [-1, -1]:
-        return True
+        return True  # Returns True if the potential spot is an "empty" list,
+        # meaning that it isn't on the game board. These lists are a byproduct of the place_ships() function
     else:
-        for n in range(5 * double):
+        for n in range(5 * multiplier):  # Checks entire ship dictionary, from "ship 1" to "ship 20"
```
Comment (RustyRaptor) [inline battleship.py:74]: Super

File: battleship.py [hunk 5]
```diff
@@ -342,7 +345,7 @@ def print_exit():  # Exits the game
                     print("\nXXXXXXXXXXXXXXX\n    Hit!!!!    \nXXXXXXXXXXXXXXX\n")
                 else:
                     print("\n///////////////\n    Miss :(    \n///////////////\n")
-                if plrs[2]["tally"] >= double * 17:
+                if plrs[2]["tally"] >= multiplier * 17:
                     print("\n------------------------------\nLmao the computer beat you. \
                     Cya nerd.\n------------------------------")
                     exit()
```
Comment (RustyRaptor) [inline battleship.py:351]: Good work, now new programmers can understand the code and contribute!!

**Review by RustyRaptor — APPROVED** [review]
Good work let's keep up the pace.

**Pull request merged**